org.gitlab.neva_blyad.yat.yml

```diff
@@ -0,0 +1,286 @@
+#    org.gitlab.neva_blyad.yat.yml
+#    Copyright (C) 2024 НЕВСКИЙ БЛЯДИНА <neva_blyad@lovecry.pt>
+#                                       <neva_blyad@lovecri.es>
+#
+#    This file is part of yat.
+#
+#    yat is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    yat is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with yat.  If not, see <https://www.gnu.org/licenses/>.
+
+# TODO:
+# meson-perl
+# libpeas-perl
+
+app-id: org.gitlab.neva_blyad.yat
```

Comment:
`org.gitlab` isn't valid for app-id based on gitlab. It is `io.gitlab`

Reply:
Done